test/common/upstream/logical_dns_cluster_test.cc

```diff
@@ -386,7 +399,7 @@ TEST_F(LogicalDnsClusterTest, BadConfig) {
       port_value: 443
   )EOF";
 
-  EXPECT_THROW_WITH_MESSAGE(setupFromV2Yaml(multiple_hosts_yaml), EnvoyException,
+  EXPECT_THROW_WITH_MESSAGE(setupFromV3Yaml(multiple_hosts_yaml, false), EnvoyException,
```

Comment:
I don't think all tests with exceptions should avoid boosting, only those where the exception is relate to some deprecated feature.

Reply:
@htuch thanks for review. done.
 here i need to update exception message according to V3 exception output.